### ISSUE TITLE
Ctrl+X is bound to Cut selection to clipboard, but shows Execute command dialog

### ISSUE THREAD

**Issue by icyrockcom** [MeanEYE/Sunflower#433]
Not sure if I'm missing some keybinding or some configuration somewhere that enables this.

**Comment by MeanEYE** [comment]
This bug I need every help solving. In short something is broken with accelerators in new GTK+ version and I am unable to find a solution for it. Granted Sunflower generates accelerators in a bit orthodox way, but the issue doesn't lie there (I don't think). I tried writing small tests where I'd define `Ctrl+F1` and `Alt+F1` and I'd always get the same function call no matter how they are configured and connected. I tried with all kinds of different ways of connecting accelerators but to no success. If you don't mind please take a look at #364 and confirm that test I posted there works.

**Comment by icyrockcom** [comment]
Your test seems to be working fine for me if I'm reading it correctly:

```
$ python EWrAzvK5.py 
One: 65470 - <flags GDK_SHIFT_MASK of type Gdk.ModifierType>
Two: 65470 - <flags 0 of type Gdk.ModifierType>
One: 50 - <flags GDK_CONTROL_MASK of type Gdk.ModifierType>
Two: 50 - <flags 0 of type Gdk.ModifierType>
One: 65471 - <flags GDK_CONTROL_MASK of type Gdk.ModifierType>
Two: 65471 - <flags 0 of type Gdk.ModifierType>
```

This is me pressing keys in order defined in the test:

```
Shift+F1  - One: 65470 - <flags GDK_SHIFT_MASK of type Gdk.ModifierType>
F1        - Two: 65470 - <flags 0 of type Gdk.ModifierType>
Ctrl+2    - One: 50 - <flags GDK_CONTROL_MASK of type Gdk.ModifierType>
2         - Two: 50 - <flags 0 of type Gdk.ModifierType>
Ctrl + F2 - One: 65471 - <flags GDK_CONTROL_MASK of type Gdk.ModifierType>
F2        - Two: 65471 - <flags 0 of type Gdk.ModifierType>
```

I'm on xubuntu if that matters.

**Comment by icyrockcom** [comment]
Interesting though that the test is working, but the app doesn't (in the same x11 environment). That leads me to believe either 1) the app is handling the accelerators in a different way or 2) the issue is not related to the accelerators directly.

I've looked at the code briefly and it looks to me that accelerators are in conflict with manual handling here:

https://github.com/MeanEYE/Sunflower/blob/develop/sunflower/plugin_base/item_list.py#L603-604

Now I'm not that familiar with GTK, but looking at [their docs](https://developer.gnome.org/gtkmm-tutorial/stable/sec-keyboardevents-propagation.html.en):

> Contrary to other events, keyboard events are first sent to the toplevel window (Gtk::Window), where it will be checked for any keyboard shortcuts that may be set (accelerator keys and mnemonics, used for selecting menu items from the keyboard). After this (and assuming the event wasn't handled), it is sent to the widget which has focus, and the propagation begins from there. 

that shouldn't be the case, so there might be something else in play.

For now, I've just commented the above two lines out and made it `return False`, which makes <kbd>Ctrl+X</kbd> work again.
